quantum/process_keycode/process_unicode_common.c

```diff
@@ -32,6 +33,14 @@ uint8_t get_unicode_input_mode(void) {
   return input_mode;
```

Comment:
@drashna Is a call to `initialize_unicode_input_mode` needed here?

Reply:
shouldn't be.  By the time this is called anywhere, the mode should have been read from eeprom.

Reply:
Alright, I thought I'd check. Even better this way.